frontend/nc-games/src/components/Categories.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from "react";
+import * as api from "../api";
+import Category from "./Category";
+export default function Categories() {
+  const [categories, setCategories] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
you don't need to set isLoading to true here as it is already set on line 6

Reply:
Agree with Heather again.